avocado/plugins/run.py

```diff
@@ -77,6 +77,9 @@ def configure(self, parser):
                                   'You can also use suffixes, like: '
                                   ' s (seconds), m (minutes), h (hours). '))
 
+        parser.add_argument('--failfast', action='store_true',
+                            help='Exit job on first failed test.')
```

Comment:
Minor suggestion: let's use the same wording.  In this case: "Interrupt job on first failed test".